<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>305</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="btn_clear">
        <property name="text">
         <string>清除</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btn_QTextStream_write">
        <property name="text">
         <string>文本文件写入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_QTextStream_read">
        <property name="text">
         <string>文本文件读出</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="cBox_Codec"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btn_QDataStream_write">
        <property name="text">
         <string>二进制写入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_QDataStream_read">
        <property name="text">
         <string>二进制读取</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_QFile_write">
        <property name="text">
         <string>QFile写入</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_QFile_read">
        <property name="text">
         <string>QFile读取</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QPushButton" name="btn_creatdir">
        <property name="text">
         <string>创建文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_dirinfo">
        <property name="text">
         <string>遍历文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_Watch">
        <property name="text">
         <string>文件夹监控</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_QFileInfo_file">
        <property name="text">
         <string>文件信息</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>386</width>
     <height>28</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
